src/test/java/homeWorks/ttykhovetska/JUnitTestNGHomeWork.java

```diff
@@ -0,0 +1,92 @@
+package homeWorks.ttykhovetska;
+
+import atqc.junit_testng.TestBase;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class JUnitTestNGHomeWork {
+
+    public String baseUrl="https://try.test.io/test-io-platform/";
+    public String driverPath = "/opt/homebrew/bin/chromedriver";
+    public WebDriver driver;
+    private static final String User_First_Name = "David";
+    private static final String User_Last_Name = "Dakhovny";
+    private static final String User_Company = "Marsik";
+    private static final String User_EMail = "moonlight@gmail.com";
+
+    public String test_Form_User_First_Name = "//input[@id='first_name']";
+    public String test_Form_User_Last_Name = "//input[@id='last_name']";
+    public String test_Form_User_Company = "//input[@id='company']";
+    public String test_Form_User_EMail = "//input[@type='email']";
+    public String try_test_IO_Button = " //button[@id='lp-pom-button-10']";
+    public String pop_Up_Button = "//div[@class=\"lp-pom-block-content\"]";
+    public String cross_Button = "//button[contains(text(),'Close')]";
+
+
+
+    @BeforeClass
+
+    public void beforeJUnitTestNGHomeWork(){
+        System.setProperty("webdriver.chrome.driver", driverPath);
+        driver=new ChromeDriver();
+        driver.get(baseUrl);
+
+    }
+    private static final String STEP_1 = "Fill all inputs";
+
+    @Test
+    public void fillingInTestForm(){
+     driver.findElement(By.xpath(test_Form_User_First_Name)).sendKeys(User_First_Name);
+        driver.findElement(By.xpath(test_Form_User_Last_Name)).sendKeys(User_Last_Name);
+        driver.findElement(By.xpath(test_Form_User_Company)).sendKeys(User_Company);
+        driver.findElement(By.xpath(test_Form_User_EMail)).sendKeys(User_EMail);
+        driver.findElement(By.xpath(try_test_IO_Button)).click();
+    }
+    private static final String Step_2 = "Check pop up text";
+
+    @Test
+    public void checkingPopUpText(){
+        Assert.assertTrue(driver.findElement(By.xpath(pop_Up_Button)).getText().equals("Thank You! We'll get back to you soon."));
+    }
+
+    @AfterClass
+
+    public void afterJUnitTestNGHomeWork(){
```

Comment:
after class failed, if second step doesn't work. because it tries to close the button that not appear. please, add "if" statement with checking if button display.